test/pensions/config/servicePeriods.unit.spec.jsx

```diff
@@ -38,6 +38,52 @@ describe('Pensions service periods', () => {
     expect(onSubmit.called).to.be.false;
   });
 
+  it('should display warning if the veteran did not serve during a wartime period', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = getFormDOM(form);
+
+    formDOM.fillData('#root_servicePeriods_0_serviceBranch', 'Army');
+    formDOM.fillData('#root_servicePeriods_0_activeServiceDateRange_toMonth', '1');
```

Comment:
You can use `formDOM.fillDate('root_servicePeriods_0_activeServiceDateRange_to', '1-1-1983');` for this. (Though perhaps we should change the format of that to 'YYYY-MM-DD'...would require some refactoring, though.)